### PR TITLE
remove @NotBlank annotation

### PR DESCRIPTION
**Pull request by PAKETA-0629**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/main/java/com/softserve/teachua/dto/location/LocationProfile.java [hunk 1]
```diff
@@ -33,10 +33,10 @@
     @NotBlank
     @CheckRussian
     private String cityName;
-    @NotBlank
+
```
Comment (PAKETA-0629) [inline src/main/java/com/softserve/teachua/dto/location/LocationProfile.java:36]: remove NotBlank because districtName and stationName is optional fields

**Review by VictoriaAmazonka — APPROVED** [review]

File: src/main/java/com/softserve/teachua/dto/location/LocationProfile.java (hunk 1 above)
Comment (OH72) [reply src/main/java/com/softserve/teachua/dto/location/LocationProfile.java:36]: Why do you think so?
When I create a club or center, I need to select or create a location. And when I choose to create, I can't finish creating a location without these fields.

**Review by VictoriaAmazonka — APPROVED** [review]

File: src/main/java/com/softserve/teachua/dto/location/LocationProfile.java (hunk 1 above)
Comment (PAKETA-0629) [reply src/main/java/com/softserve/teachua/dto/location/LocationProfile.java:36]: UI allows to create location without district and station. service layer also allows you to create a location with these empty fields

Comment (OH72) [reply src/main/java/com/softserve/teachua/dto/location/LocationProfile.java:36]: I checked it again. Yes, right, sorry.

**Pull request merged**